package com.yash.ecommerce_backend.auth.dto;

import jakarta.validation.constraints.Email;
import jakarta.validation.constraints.NotBlank;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class LoginRequest {
    @NotBlank(message = "Email is Required")
    @Email(message = "Invalid email format")
    private String email;
    @NotBlank(message = "Password is Required")
    private String password;
}
